cranelift/codegen/src/machinst/vcode.rs

```diff
@@ -523,8 +562,24 @@ impl<I: VCodeInst> VCode<I> {
 
             let (start, end) = self.block_ranges[block as usize];
             for iix in start..end {
+                let srcloc = self.srclocs[iix as usize];
+                if srcloc != cur_srcloc {
+                    if !cur_srcloc.is_default() {
+                        code_section.end_srcloc();
+                    }
+                    if !srcloc.is_default() {
```

Comment:
I actually want to know about all instructions, not just those with a non-default srcloc.

Reply:
Thanks for pointing this out -- you're right! New PR here: #1632